### ISSUE TITLE
Add change self password UI for  User [Feature request]

### ISSUE THREAD

**Issue by ameygat** [floragunncom/search-guard-kibana-plugin#99]
I think the basic password management of changing own password would be very useful feature in the Kibana Plugin. Right now I am unable to see any option that will allow users to change their password from Kibana User interface.

**Comment by floragunncom** [comment]
True, this feature is not there. The main reason is that it would only work if you use the internal user database to store users. For all other authentication methods like LDAP, Active Directory, JSON web token, certificate based auth and so on it would not work since the user credentials are stored on an external system. We'll keep this issue open for the moment and investigate how much effort it is to implement this for the internal user database.
